contrib/localnet/orchestrator/start-zetae2e.sh

```diff
@@ -10,6 +10,8 @@
 # Trap signals and forward to children
 trap 'kill -- -$$' SIGINT SIGTERM
 
+/usr/sbin/sshd
```

Comment:
What is sshing into the orchestrator?

Reply:
removed,
I was using it earlier to push the private keys to the orchestrator, but it's not needed now as I am pulling them instead